### PR TITLE
RDSSecurityGroupIngressOpenToWorldRule

### PR DESCRIPTION
**Pull request by jsoucheiron**
### Additions
- New rule: `RDSSecurityGroupIngressOpenToWorldRule`
### Updates
- Bumped minimum `pycfmodel` version to `0.20.0`

### CODE REVIEW

File: cfripper/rules/rds_security_group.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+from pycfmodel.model.resources.security_group_ingress import RDSDBSecurityGroupIngress
+
+from cfripper.model.enums import RuleGranularity, RuleMode, RuleRisk
+from cfripper.model.result import Result
+from cfripper.rules import ResourceSpecificRule
+
+
+class RDSSecurityGroupIngressOpenToWorldRule(ResourceSpecificRule):
+    """
+    Checks if security groups have an ingress IP that is open to the world an RDS security group.
```
Comment (w0rmr1d3r) [inline cfripper/rules/rds_security_group.py:13]: This sentence looks a little bit confusing to me.
```suggestion
    Checks if RDS native security groups have an ingress IP that is open to the world.
```

File: cfripper/rules/rds_security_group.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        |`logical_id`             | str                                            | ID used in Cloudformation to refer the resource being analysed |
+        |`resource`               | `RDSDBSecurityGroup/RDSDBSecurityGroupIngress` | Resource that is being addressed                               |
+        |`ingress_obj`            | `DBSecurityGroupIngressProp`                   | DBSecurityGroupIngress being checked found in the Resource     |
+    """
+
+    GRANULARITY = RuleGranularity.RESOURCE
+    RESOURCE_TYPES = (RDSDBSecurityGroup, RDSDBSecurityGroupIngress)
+    RULE_MODE = RuleMode.BLOCKING
+    RISK_VALUE = RuleRisk.HIGH
+    REASON = "RDS DB Security group policy {} should not have ingress open to the world"
```
Comment (w0rmr1d3r) [inline cfripper/rules/rds_security_group.py:63]:
```suggestion
    REASON = "RDS DB Security group {} should not have ingress open to the world"
```

**Review by w0rmr1d3r — COMMENTED** [review]
Looks great, some suggestions to make it easier to understand why this can get flagged

**Review by w0rmr1d3r — APPROVED** [review]
nice! this could be merged and released together with https://github.com/Skyscanner/cfripper/pull/225

**Review by ocrawford555 — APPROVED** [review]

**Pull request merged**